### PR TITLE
[forge] stable 3 times a week

### PR DESCRIPTION
**Pull request by rustielin**
## Type of Change
- [ ] New feature
- [ ] Bug fix
- [ ] Breaking change
- [ ] Performance improvement
- [ ] Refactoring
- [ ] Dependency update
- [ ] Documentation update
- [x] Tests

## Which Components or Systems Does This Change Impact?
- [ ] Validator Node
- [ ] Full Node (API, Indexer, etc.)
- [ ] Move/Aptos Virtual Machine
- [ ] Aptos Framework
- [ ] Aptos CLI/SDK
- [x] Developer Infrastructure
- [ ] Other (specify)

## How Has This Been Tested?

Nightly CI run on main after this lands

## Key Areas to Review

This PR makes `forge-stable` match the cadence of `replay-verify`: Sun, Tues, Thurs at 22:00 UTC. This allows us to have release candidate Mon, Wed, Fri, and also ensures that both of these nightly tests only test the same RC

## Checklist
- [x] I have read and followed the [CONTRIBUTING](https://github.com/aptos-labs/aptos-core/blob/main/CONTRIBUTING.md) doc
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I identified and added all stakeholders and component owners affected by this change as reviewers
- [x] I tested both happy and unhappy path of the functionality
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

**Review by perryjrandall — APPROVED** [review]

File: .github/workflows/forge-stable.yaml [hunk 1]
```diff
@@ -136,7 +133,7 @@ jobs:
 
   run-forge-realistic-env-max-load-long:
     if: ${{ github.event_name != 'pull_request' && always() }}
-    needs: [ determine-test-metadata, run-forge-framework-upgrade-test ] # Only run after the previous job completes
+    needs: [determine-test-metadata, run-forge-framework-upgrade-test] # Only run after the previous job completes
```
Comment (bchocho) [inline .github/workflows/forge-stable.yaml:136]: Somewhat unrelated, but do we need some style for yaml files?

**Review by bchocho — APPROVED** [review]

File: .github/workflows/forge-stable.yaml (hunk 1 above)
Comment (rustielin) [reply .github/workflows/forge-stable.yaml:136]: it's not enforced anywhere (though it probably should). I use prettier on file save

Comment (bchocho) [reply .github/workflows/forge-stable.yaml:136]: IIRC my IntelliJ default adds the spaces 🤦

Comment (sionescu) [reply .github/workflows/forge-stable.yaml:136]: We should use prettier everywhere.

**Review by sionescu — APPROVED** [review]

**Pull request merged**